continuous_integration/install.sh

```diff
@@ -21,11 +28,23 @@ if [[ "$DISTRIB" == "conda" ]]; then
 
     # Use the miniconda installer for faster download / install of conda
     # itself
-    wget http://repo.continuum.io/miniconda/Miniconda-3.6.0-Linux-x86_64.sh \
-        -O miniconda.sh
+    pushd .
+    cd
+    mkdir -p download
+    cd download
+    echo "Cached in $HOME/download :"
+    ls -l
+    echo
```

Comment:
It is a new line yes.
This permits to see if the download folder was cached or not. 
I let it there so people can get what is going on with travys (as several changes are being done) 
I can push a cleaner version without the logs if you want.